### PR TITLE
Fix pip install

### PR DESCRIPTION
**Pull request by shouldsee**
use `setuptools.setup(setup_requires=[”numpy“])` to  specify build-time dependency.

Feel free to test this without disutils.

### CODE REVIEW

**Comment by brentp** [comment]
this looks fine to me. can someone else (@tanghaibao ?) take a look?
@shouldsee please remove the code rather than commenting it out.

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,27 @@
+version = '0.1.5'
```
Comment (tanghaibao) [inline setup.py]: Move version var after the imports? Per [PEP8](https://www.python.org/dev/peps/pep-0008/):

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

File: setup.py [hunk 2]
```diff
@@ -1,8 +1,27 @@
...
+
 from setuptools import setup
 from setuptools.extension import Extension
+from setuptools.command.build_ext import build_ext
+
+class _build_ext(build_ext):
+    """build_ext command for use when numpy headers are needed.
+    
+    https://stackoverflow.com/a/42163080/8083313
+"""
```
Comment (tanghaibao) [inline setup.py]: Maybe indent this """?

**Comment by tanghaibao** [comment]
Just some minor stylistic suggestion, but otherwise good to go. Thanks -

File: setup.py (hunk 2 above)
Comment (shouldsee) [reply setup.py]: fixed

File: setup.py (hunk 1 above)
Comment (shouldsee) [reply setup.py]: fixed

**Comment by brentp** [comment]
thanks @shouldsee and Haibao

**Pull request merged**